Build failure: tectonic-unwrapped

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

`nix-build -A tectonic-unwrapped` in local clone of nixpkgs (nixos-unstable or master)

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/293104013

### Relevant log output

```console
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/wm1k8y42lw9v17z2llypkh7ad3ah6dvy-source
source root is source
Executing cargoSetupPostUnpackHook
Finished cargoSetupPostUnpackHook
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
applying patch /nix/store/h1gxn7d6p4gx6gxiz8jsmzwr6lhwbvd0-1202-fix-build-with-rust-1_80
patching file Cargo.lock
Hunk #1 succeeded at 2359 (offset -274 lines).
Hunk #2 succeeded at 2683 (offset -252 lines).
Hunk #3 succeeded at 2706 (offset -252 lines).
patching file Cargo.toml
Hunk #1 succeeded at 97 with fuzz 2.
applying patch /nix/store/h4jarfpjv11pniijqamirda2d96h9c8k-1155-fix-endless-reruns-when-generating-bbl
patching file crates/engine_bibtex/src/lib.rs
Executing cargoSetupPostPatchHook
Validating consistency between /build/source/Cargo.lock and /build/tectonic-0.15.0-vendor/Cargo.lock
Finished cargoSetupPostPatchHook
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Executing cargoBuildHook
cargoBuildHook flags: -j 16 --target x86_64-unknown-linux-gnu --offline --profile release --features=external-harfbuzz
[1m[32m   Compiling[0m libc v0.2.153
[1m[32m   Compiling[0m proc-macro2 v1.0.78
[1m[32m   Compiling[0m unicode-ident v1.0.12
[1m[32m   Compiling[0m cfg-if v1.0.0
[1m[32m   Compiling[0m pkg-config v0.3.29
[1m[32m   Compiling[0m vcpkg v0.2.15
[1m[32m   Compiling[0m autocfg v1.1.0
[1m[32m   Compiling[0m thiserror v1.0.56
[1m[32m   Compiling[0m memchr v2.7.1
[1m[32m   Compiling[0m once_cell v1.19.0
[1m[32m   Compiling[0m version_check v0.9.4
[1m[32m   Compiling[0m log v0.4.20
[1m[32m   Compiling[0m bitflags v2.4.2
[1m[32m   Compiling[0m itoa v1.0.10
[1m[32m   Compiling[0m pin-project-lite v0.2.13
[1m[32m   Compiling[0m regex-syntax v0.8.2
[1m[32m   Compiling[0m same-file v1.0.6
[1m[32m   Compiling[0m pin-utils v0.1.0
[1m[32m   Compiling[0m bytes v1.5.0
[1m[32m   Compiling[0m walkdir v2.4.0
[1m[32m   Compiling[0m ppv-lite86 v0.2.17
[1m[32m   Compiling[0m typenum v1.17.0
[1m[32m   Compiling[0m futures-core v0.3.30
[1m[32m   Compiling[0m siphasher v0.3.11
[1m[32m   Compiling[0m tectonic_dep_support v0.1.1 (/build/source/crates/dep_support)
[1m[32m   Compiling[0m generic-array v0.14.7
[1m[32m   Compiling[0m slab v0.4.9
[1m[32m   Compiling[0m phf_shared v0.11.2
[1m[32m   Compiling[0m futures-sink v0.3.30
[1m[32m   Compiling[0m num-traits v0.2.17
[1m[32m   Compiling[0m serde v1.0.196
[1m[32m   Compiling[0m adler v1.0.2
[1m[32m   Compiling[0m anyhow v1.0.79
[1m[32m   Compiling[0m crossbeam-utils v0.8.19
[1m[32m   Compiling[0m bitflags v1.3.2
[1m[32m   Compiling[0m tinyvec_macros v0.1.1
[1m[32m   Compiling[0m aho-corasick v1.1.2
[1m[32m   Compiling[0m tinyvec v1.6.0
[1m[32m   Compiling[0m miniz_oxide v0.7.2
[1m[32m   Compiling[0m futures-channel v0.3.30
[1m[32m   Compiling[0m futures-io v0.3.30
[1m[32m   Compiling[0m rustix v0.38.31
[1m[32m   Compiling[0m quote v1.0.35
[1m[32m   Compiling[0m futures-task v0.3.30
[1m[32m   Compiling[0m syn v2.0.48
[1m[32m   Compiling[0m crc32fast v1.3.2
[1m[32m   Compiling[0m linux-raw-sys v0.4.13
[1m[32m   Compiling[0m smallvec v1.13.1
[1m[32m   Compiling[0m fnv v1.0.7
[1m[32m   Compiling[0m hex v0.4.3
[1m[32m   Compiling[0m lazy_static v1.4.0
[1m[32m   Compiling[0m tectonic_cfg_support v0.1.4 (/build/source/crates/cfg_support)
[1m[32m   Compiling[0m unicode-normalization v0.1.22
[1m[32m   Compiling[0m tectonic_errors v0.2.1 (/build/source/crates/errors)
[1m[32m   Compiling[0m tracing-core v0.1.32
[1m[32m   Compiling[0m sha1_smol v1.0.0
[1m[32m   Compiling[0m cc v1.0.83
[1m[32m   Compiling[0m mio v0.8.10
[1m[32m   Compiling[0m getrandom v0.2.12
[1m[32m   Compiling[0m num_cpus v1.16.0
[1m[32m   Compiling[0m signal-hook-registry v1.4.1
[1m[32m   Compiling[0m socket2 v0.5.5
[1m[32m   Compiling[0m rand_core v0.6.4
[1m[32m   Compiling[0m crypto-common v0.1.6
[1m[32m   Compiling[0m block-buffer v0.10.4
[1m[32m   Compiling[0m lock_api v0.4.11
[1m[32m   Compiling[0m powerfmt v0.2.0
[1m[32m   Compiling[0m rand_chacha v0.3.1
[1m[32m   Compiling[0m digest v0.10.7
[1m[32m   Compiling[0m time-core v0.1.2
[1m[32m   Compiling[0m regex-automata v0.4.5
[1m[32m   Compiling[0m fastrand v2.0.1
[1m[32m   Compiling[0m minimal-lexical v0.2.1
[1m[32m   Compiling[0m rand v0.8.5
[1m[32m   Compiling[0m home v0.5.9
[1m[32m   Compiling[0m num-conv v0.1.0
[1m[32m   Compiling[0m parking_lot_core v0.9.9
[1m[32m   Compiling[0m nom v7.1.3
[1m[32m   Compiling[0m time-macros v0.2.18
[1m[32m   Compiling[0m deranged v0.3.11
[1m[32m   Compiling[0m tectonic_status_base v0.2.1 (/build/source/crates/status_base)
[1m[32m   Compiling[0m memoffset v0.7.1
[1m[32m   Compiling[0m num_threads v0.1.6
[1m[32m   Compiling[0m hashbrown v0.14.3
[1m[32m   Compiling[0m xdg v2.5.2
[1m[32m   Compiling[0m libz-sys v1.1.15
[1m[32m   Compiling[0m tectonic_bridge_core v0.4.1 (/build/source/crates/bridge_core)
[1m[32m   Compiling[0m openssl-sys v0.9.99
[1m[32m   Compiling[0m equivalent v1.0.1
[1m[32m   Compiling[0m unicode-width v0.1.11
[1m[32m   Compiling[0m phf_generator v0.11.2
[1m[32m   Compiling[0m phf_codegen v0.11.2
[1m[32m   Compiling[0m scopeguard v1.2.0
[1m[32m   Compiling[0m cpufeatures v0.2.12
[1m[32m   Compiling[0m app_dirs2 v2.5.5
[1m[32m   Compiling[0m flate2 v1.0.28
[1m[32m   Compiling[0m sha2 v0.10.8
[1m[32m   Compiling[0m tempfile v3.9.0
[1m[32m   Compiling[0m indexmap v2.2.2
[1m[32m   Compiling[0m crossbeam-epoch v0.9.18
[1m[32m   Compiling[0m tectonic_bridge_graphite2 v0.2.2 (/build/source/crates/bridge_graphite2)
[1m[32m   Compiling[0m tectonic_bridge_flate v0.1.7 (/build/source/crates/bridge_flate)
[1m[32m   Compiling[0m gix-trace v0.1.7
[1m[32m   Compiling[0m crossbeam-deque v0.8.5
[1m[32m   Compiling[0m parking_lot v0.12.1
[1m[32m   Compiling[0m phf v0.11.2
[1m[32m   Compiling[0m btoi v0.4.3
[1m[32m   Compiling[0m time v0.3.36
[1m[32m   Compiling[0m openssl v0.10.63
[1m[32m   Compiling[0m foreign-types-shared v0.1.1
[1m[32m   Compiling[0m percent-encoding v2.3.1
[1m[32m   Compiling[0m foreign-types v0.3.2
[1m[32m   Compiling[0m nix v0.26.4
[1m[32m   Compiling[0m tectonic_bridge_harfbuzz v0.2.9 (/build/source/crates/bridge_harfbuzz)
[1m[32m   Compiling[0m tectonic_pdf_io v0.4.1 (/build/source/crates/pdf_io)
[1m[32m   Compiling[0m gix-utils v0.1.9
[1m[32m   Compiling[0m md-5 v0.10.6
[1m[32m   Compiling[0m http v0.2.11
[1m[32m   Compiling[0m tectonic_bridge_freetype2 v0.2.0 (/build/source/crates/bridge_freetype2)
[1m[32m   Compiling[0m tectonic_bridge_icu v0.2.1 (/build/source/crates/bridge_icu)
[1m[32m   Compiling[0m ryu v1.0.16
[1m[32m   Compiling[0m native-tls v0.2.11
[1m[32m   Compiling[0m httparse v1.8.0
[1m[32m   Compiling[0m terminfo v0.8.0
[1m[32m   Compiling[0m inotify-sys v0.1.5
[1m[32m   Compiling[0m dirs-sys v0.3.7
[1m[32m   Compiling[0m memmap2 v0.5.10
[1m[32m   Compiling[0m ucd-trie v0.1.6
[1m[32m   Compiling[0m async-trait v0.1.77
[1m[32m   Compiling[0m try-lock v0.2.5
[1m[32m   Compiling[0m openssl-probe v0.1.5
[1m[32m   Compiling[0m want v0.3.1
[1m[32m   Compiling[0m http-body v0.4.6
[1m[32m   Compiling[0m dirs v4.0.0
[1m[32m   Compiling[0m bstr v1.9.0
[1m[32m   Compiling[0m inotify v0.9.6
[1m[32m   Compiling[0m form_urlencoded v1.2.1
[1m[32m   Compiling[0m tectonic_xetex_layout v0.2.4 (/build/source/crates/xetex_layout)
[1m[32m   Compiling[0m gix-sec v0.8.4
[1m[32m   Compiling[0m filetime v0.2.23
[1m[32m   Compiling[0m crossbeam-channel v0.5.11
[1m[32m   Compiling[0m nibble_vec v0.1.0
[1m[32m   Compiling[0m proc-macro-error-attr v1.0.4
[1m[32m   Compiling[0m unicode-bom v2.0.3
[1m[32m   Compiling[0m syn v1.0.109
[1m[32m   Compiling[0m tower-service v0.3.2
[1m[32m   Compiling[0m globset v0.4.14
[1m[32m   Compiling[0m unic-common v0.9.0
[1m[32m   Compiling[0m libm v0.2.8
[1m[32m   Compiling[0m either v1.9.0
[1m[32m   Compiling[0m unic-char-range v0.9.0
[1m[32m   Compiling[0m endian-type v0.1.2
[1m[32m   Compiling[0m unicode-bidi v0.3.15
[1m[32m   Compiling[0m httpdate v1.0.3
[1m[32m   Compiling[0m radix_trie v0.2.1
[1m[32m   Compiling[0m thiserror-impl v1.0.56
[1m[32m   Compiling[0m tokio-macros v2.2.0
[1m[32m   Compiling[0m serde_derive v1.0.196
[1m[32m   Compiling[0m futures-macro v0.3.30
[1m[32m   Compiling[0m tracing-attributes v0.1.27
[1m[32m   Compiling[0m openssl-macros v0.1.1
[1m[32m   Compiling[0m miette-derive v5.10.0
[1m[32m   Compiling[0m tokio v1.36.0
[1m[32m   Compiling[0m ignore v0.4.22
[1m[32m   Compiling[0m futures-util v0.3.30
[1m[32m   Compiling[0m gix-hash v0.11.4
[1m[32m   Compiling[0m gix-date v0.5.1
[1m[32m   Compiling[0m gix-features v0.29.0
[1m[32m   Compiling[0m gix-path v0.8.4
[1m[32m   Compiling[0m tracing v0.1.40
[1m[32m   Compiling[0m gix-fs v0.1.1
[1m[32m   Compiling[0m tectonic_io_base v0.4.3 (/build/source/crates/io_base)
[1m[32m   Compiling[0m gix-tempfile v5.0.3
[1m[32m   Compiling[0m regex v1.10.3
[1m[32m   Compiling[0m gix-actor v0.20.0
[1m[32m   Compiling[0m gix-validate v0.7.7
[1m[32m   Compiling[0m gix-object v0.29.2
[1m[32m   Compiling[0m gix-lock v5.0.1
[1m[32m   Compiling[0m parse-zoneinfo v0.3.0
[1m[32m   Compiling[0m miette v5.10.0
[1m[32m   Compiling[0m chrono-tz-build v0.2.1
[1m[32m   Compiling[0m gix-glob v0.7.0
[1m[32m   Compiling[0m gix-ref v0.29.1
[1m[32m   Compiling[0m gix-config-value v0.12.5
[1m[32m   Compiling[0m pest v2.7.6
[1m[32m   Compiling[0m chrono-tz v0.8.5
[1m[32m   Compiling[0m idna v0.5.0
[1m[32m   Compiling[0m watchexec-signals v1.0.0
[1m[32m   Compiling[0m futures-executor v0.3.30
[1m[32m   Compiling[0m futures v0.3.30
[1m[32m   Compiling[0m notify v5.2.0
[1m[32m   Compiling[0m unic-char-property v0.9.0
[1m[32m   Compiling[0m which v4.4.2
[1m[32m   Compiling[0m gix-config v0.22.0
[1m[32m   Compiling[0m pest_meta v2.7.6
[1m[32m   Compiling[0m unic-ucd-version v0.9.0
[1m[32m   Compiling[0m proc-macro-error v1.0.4
[1m[32m   Compiling[0m dunce v1.0.4
[1m[32m   Compiling[0m iana-time-zone v0.1.60
[1m[32m   Compiling[0m serde_json v1.0.113
[1m[32m   Compiling[0m tokio-util v0.7.10
[1m[32m   Compiling[0m tokio-stream v0.1.14
[1m[32m   Compiling[0m tokio-native-tls v0.3.1
[1m[32m   Compiling[0m base64 v0.21.7
[1m[32m   Compiling[0m project-origins v1.2.0
[1m[32m   Compiling[0m h2 v0.3.24
[1m[32m   Compiling[0m byteorder v1.5.0
[1m[32m   Compiling[0m event-listener v2.5.3
[1m[32m   Compiling[0m async-priority-channel v0.1.0
[1m[32m   Compiling[0m ignore-files v1.3.1
[1m[32m   Compiling[0m rustls-pemfile v1.0.4
[1m[32m   Compiling[0m serde_urlencoded v0.7.1
[1m[32m   Compiling[0m toml_datetime v0.6.5
[1m[32m   Compiling[0m serde_spanned v0.6.5
[1m[32m   Compiling[0m pest_generator v2.7.6
[1m[32m   Compiling[0m chrono v0.4.33
[1m[32m   Compiling[0m command-group v2.1.0
[1m[32m   Compiling[0m unic-ucd-segment v0.9.0
[1m[32m   Compiling[0m watchexec-events v1.0.0
[1m[32m   Compiling[0m url v2.5.0
[1m[32m   Compiling[0m async-recursion v1.0.5
[1m[32m   Compiling[0m clearscreen v2.0.1
[1m[32m   Compiling[0m tectonic_engine_xdvipdfmx v0.4.1 (/build/source/crates/engine_xdvipdfmx)
[1m[32m   Compiling[0m backtrace v0.3.69
[1m[32m   Compiling[0m tectonic_engine_xetex v0.4.4 (/build/source/crates/engine_xetex)
[0m[1m[33mwarning[0m[0m[1m: unexpected `cfg` condition value: `profile`[0m
[0m  [0m[0m[1m[38;5;12m--> [0m[0mcrates/engine_xetex/build.rs:28:46[0m
[0m   [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m28[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    const PROFILE_BUILD_ENABLED: bool = cfg!(feature = "profile");[0m
[0m   [0m[0m[1m[38;5;12m|[0m[0m                                              [0m[0m[1m[33m^^^^^^^^^^^^^^^^^^^[0m
[0m   [0m[0m[1m[38;5;12m|[0m
[0m   [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: expected values for `feature` are: `external-harfbuzz`[0m
[0m   [0m[0m[1m[38;5;12m= [0m[0m[1mhelp[0m[0m: consider adding `profile` as a feature in `Cargo.toml`[0m
[0m   [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration[0m
[0m   [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: `#[warn(unexpected_cfgs)]` on by default[0m

[1m[32m   Compiling[0m encoding_rs v0.8.33
[1m[32m   Compiling[0m winnow v0.5.37
[1m[33mwarning[0m[1m:[0m `tectonic_engine_xetex` (build script) generated 1 warning
[1m[32m   Compiling[0m gimli v0.28.1
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: cc1plus: warning: /nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include: No such file or directory [-Wmissing-include-dirs]
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: cc1plus: warning: /nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include: No such file or directory [-Wmissing-include-dirs]
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: cc1plus: warning: /nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include: No such file or directory [-Wmissing-include-dirs]
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: In file included from /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/ubidi.h:26,
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:                  from layout/xetex-XeTeXLayoutInterface.cpp:36:
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:561:26: error: 'auto' parameter not permitted in this context
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   561 | template <typename Type, auto closeFunction>
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                          ^~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:573:76: error: template argument 2 is invalid
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   573 |     explicit LocalOpenPointer(std::unique_ptr<Type, decltype(closeFunction)> &&p)
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                                                                            ^
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:583:78: error: template argument 2 is invalid
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   583 |     LocalOpenPointer &operator=(std::unique_ptr<Type, decltype(closeFunction)> &&p) {
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                                                                              ^
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:599:59: error: template argument 2 is invalid
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   599 |     operator std::unique_ptr<Type, decltype(closeFunction)> () && {
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                                                           ^
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/ubidi.h:579:1: note: invalid template non-type parameter
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   579 | U_DEFINE_LOCAL_OPEN_POINTER(LocalUBiDiPointer, UBiDi, ubidi_close);
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
[1m[31merror[0m[1m:[0m failed to run custom build command for `tectonic_xetex_layout v0.2.4 (/build/source/crates/xetex_layout)`

Caused by:
  process didn't exit successfully: `/build/source/target/release/build/tectonic_xetex_layout-25215a43ebd3f08c/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-env-changed=TECTONIC_DEP_BACKEND
  cargo:rerun-if-env-changed=TECTONIC_PKGCONFIG_FORCE_SEMI_STATIC
  cargo:rerun-if-env-changed=FONTCONFIG_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=SYSROOT
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-changed=layout/xetex-XeTeXFontInst.cpp
  cargo:rerun-if-changed=layout/xetex-XeTeXFontMgr.cpp
  cargo:rerun-if-changed=layout/xetex-XeTeXLayoutInterface.cpp
  cargo:rerun-if-changed=layout/xetex-XeTeXFontMgr_FC.cpp
  TARGET = Some("x86_64-unknown-linux-gnu")
  OPT_LEVEL = Some("3")
  HOST = Some("x86_64-unknown-linux-gnu")
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some("false")
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  running: "/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "layout" "-I" "/build/source/crates/bridge_core/support" "-I" "/nix/store/nziailpyy0d90vq5pi1kckbxzvjcg5ng-fontconfig-2.16.0-dev/include" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/r5iajf7yh32jw3c0hmnw0iwyx5lxhky0-harfbuzz-10.2.0-dev/include/harfbuzz" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include" "-I" "/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include" "-Wall" "-Wextra" "-Wall" "-std=c++14" "-Wall" "-Wdate-time" "-Wendif-labels" "-Wextra" "-Wformat=2" "-Wlogical-op" "-Wmissing-declarations" "-Wmissing-include-dirs" "-Wpointer-arith" "-Wredundant-decls" "-Wsuggest-attribute=noreturn" "-Wsuggest-attribute=format" "-Wshadow" "-Wswitch-bool" "-Wundef" "-Wextra-semi" "-Wno-unused-parameter" "-Wno-implicit-fallthrough" "-fno-exceptions" "-fno-rtti" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/tectonic_xetex_layout-a8004e69e698997a/out/layout/xetex-XeTeXFontInst.o" "-c" "layout/xetex-XeTeXFontInst.cpp"
  cargo:warning=cc1plus: warning: /nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include: No such file or directory [-Wmissing-include-dirs]

  exit status: 0
  running: "/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "layout" "-I" "/build/source/crates/bridge_core/support" "-I" "/nix/store/nziailpyy0d90vq5pi1kckbxzvjcg5ng-fontconfig-2.16.0-dev/include" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/r5iajf7yh32jw3c0hmnw0iwyx5lxhky0-harfbuzz-10.2.0-dev/include/harfbuzz" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include" "-I" "/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include" "-Wall" "-Wextra" "-Wall" "-std=c++14" "-Wall" "-Wdate-time" "-Wendif-labels" "-Wextra" "-Wformat=2" "-Wlogical-op" "-Wmissing-declarations" "-Wmissing-include-dirs" "-Wpointer-arith" "-Wredundant-decls" "-Wsuggest-attribute=noreturn" "-Wsuggest-attribute=format" "-Wshadow" "-Wswitch-bool" "-Wundef" "-Wextra-semi" "-Wno-unused-parameter" "-Wno-implicit-fallthrough" "-fno-exceptions" "-fno-rtti" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/tectonic_xetex_layout-a8004e69e698997a/out/layout/xetex-XeTeXFontMgr.o" "-c" "layout/xetex-XeTeXFontMgr.cpp"
  cargo:warning=cc1plus: warning: /nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include: No such file or directory [-Wmissing-include-dirs]

  exit status: 0
  running: "/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "layout" "-I" "/build/source/crates/bridge_core/support" "-I" "/nix/store/nziailpyy0d90vq5pi1kckbxzvjcg5ng-fontconfig-2.16.0-dev/include" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/r5iajf7yh32jw3c0hmnw0iwyx5lxhky0-harfbuzz-10.2.0-dev/include/harfbuzz" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include" "-I" "/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include" "-Wall" "-Wextra" "-Wall" "-std=c++14" "-Wall" "-Wdate-time" "-Wendif-labels" "-Wextra" "-Wformat=2" "-Wlogical-op" "-Wmissing-declarations" "-Wmissing-include-dirs" "-Wpointer-arith" "-Wredundant-decls" "-Wsuggest-attribute=noreturn" "-Wsuggest-attribute=format" "-Wshadow" "-Wswitch-bool" "-Wundef" "-Wextra-semi" "-Wno-unused-parameter" "-Wno-implicit-fallthrough" "-fno-exceptions" "-fno-rtti" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/tectonic_xetex_layout-a8004e69e698997a/out/layout/xetex-XeTeXLayoutInterface.o" "-c" "layout/xetex-XeTeXLayoutInterface.cpp"
  cargo:warning=cc1plus: warning: /nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include: No such file or directory [-Wmissing-include-dirs]

  cargo:warning=In file included from /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/ubidi.h:26,

  cargo:warning=                 from layout/xetex-XeTeXLayoutInterface.cpp:36:

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:561:26: error: 'auto' parameter not permitted in this context

  cargo:warning=  561 | template <typename Type, auto closeFunction>

  cargo:warning=      |                          ^~~~

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:573:76: error: template argument 2 is invalid

  cargo:warning=  573 |     explicit LocalOpenPointer(std::unique_ptr<Type, decltype(closeFunction)> &&p)

  cargo:warning=      |                                                                            ^

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:583:78: error: template argument 2 is invalid

  cargo:warning=  583 |     LocalOpenPointer &operator=(std::unique_ptr<Type, decltype(closeFunction)> &&p) {

  cargo:warning=      |                                                                              ^

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:599:59: error: template argument 2 is invalid

  cargo:warning=  599 |     operator std::unique_ptr<Type, decltype(closeFunction)> () && {

  cargo:warning=      |                                                           ^

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/ubidi.h:579:1: note: invalid template non-type parameter

  cargo:warning=  579 | U_DEFINE_LOCAL_OPEN_POINTER(LocalUBiDiPointer, UBiDi, ubidi_close);

  cargo:warning=      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~

  exit status: 1

  --- stderr


  error occurred: Command "/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "layout" "-I" "/build/source/crates/bridge_core/support" "-I" "/nix/store/nziailpyy0d90vq5pi1kckbxzvjcg5ng-fontconfig-2.16.0-dev/include" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/r5iajf7yh32jw3c0hmnw0iwyx5lxhky0-harfbuzz-10.2.0-dev/include/harfbuzz" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include" "-I" "/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include" "-Wall" "-Wextra" "-Wall" "-std=c++14" "-Wall" "-Wdate-time" "-Wendif-labels" "-Wextra" "-Wformat=2" "-Wlogical-op" "-Wmissing-declarations" "-Wmissing-include-dirs" "-Wpointer-arith" "-Wredundant-decls" "-Wsuggest-attribute=noreturn" "-Wsuggest-attribute=format" "-Wshadow" "-Wswitch-bool" "-Wundef" "-Wextra-semi" "-Wno-unused-parameter" "-Wno-implicit-fallthrough" "-fno-exceptions" "-fno-rtti" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/tectonic_xetex_layout-a8004e69e698997a/out/layout/xetex-XeTeXLayoutInterface.o" "-c" "layout/xetex-XeTeXLayoutInterface.cpp" with args "c++" did not execute successfully (status code exit status: 1).


[1m[33mwarning[0m[1m:[0m build failed, waiting for other jobs to finish...
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-mpost.c: In function 'mps_do_page':
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-mpost.c:1510:3: warning: ignoring return value of 'fread' declared with attribute 'warn_unused_result' [-Wunused-result]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:  1510 |   fread(buffer, sizeof(char), size, image_file);
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-pdfcolor.c:370:11: warning: no previous declaration for 'current_fill' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   370 | pdf_color current_fill   = {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |           ^~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-pdfcolor.c:379:11: warning: no previous declaration for 'current_stroke' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   379 | pdf_color current_stroke = {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |           ^~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-pdfcolor.c:423:11: warning: no previous declaration for 'default_color' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   423 | pdf_color default_color = {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |           ^~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_misc.c:363:18: warning: no previous declaration for 'fontattrs' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   363 | struct fontattr *fontattrs = NULL;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |                  ^~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_misc.c:364:5: warning: no previous declaration for 'num_fontattrs' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   364 | int num_fontattrs = 0;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |     ^~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_misc.c:365:5: warning: no previous declaration for 'max_fontattrs' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   365 | int max_fontattrs = 0;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |     ^~~~~~~~~~~~~
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.13.7, NixOS, 25.05 (Warbler), 25.05.20250319.a84ebe2`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.26.3`
 - nixpkgs: `/nix/store/1728d3jg85mkz2w2cvk6vi74i30fn6x7-source`



### Notify maintainers

@lluchs @doronbehar @bryango

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

> I think we are already running on clang. Turns out the previous fix only delays the problem.

@bryango are we sure we are actually running on clang? The above error message says

```rust
HOST_CXX = Some("/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/c++")
```

Surely `clangStdenv` would be named `clang-wrapper`?

```
nix-repl> :b pkgs.clangStdenv.cc

This derivation produced the following outputs:
  out -> /nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7
```

I bisected to 009de0667f91d4bd46fe5a2aa70d8e1f325291da (https://github.com/NixOS/nixpkgs/pull/386191) but it's not obvious how it breaks it.

```bash
git bisect start
# status: waiting for both good and bad commits
# good: [4dadee8df69c85396be37f2c8ba8dfc8c4d586ec] Merge master into staging-next
git bisect good 4dadee8df69c85396be37f2c8ba8dfc8c4d586ec
# bad: [baefe9d8e24a390da1e0859cb9033babff67ba65] gcc: remove .la files (#389009)
git bisect bad baefe9d8e24a390da1e0859cb9033babff67ba65
# bad: [3f2403435ab47665287af5d883a848ac6a170281] python313Packages.roman-numerals-py: init at 3.1.0
git bisect bad 3f2403435ab47665287af5d883a848ac6a170281
# good: [f6e560e4b0f85ff2f19fa6072c6c3afb27ce5d94] Merge master into staging-next
git bisect good f6e560e4b0f85ff2f19fa6072c6c3afb27ce5d94
# good: [057c2a540614d40f32ff5ae5f9ce77ebfb27d71f] libvisual: drop (#388447)
git bisect good 057c2a540614d40f32ff5ae5f9ce77ebfb27d71f
# good: [ad67ac46e35feab8c55a3e9c2331cab079212f35] treewide (Python packages): Use the buildPythonPackage-provided locale specification whenever appropriate (#383766)
git bisect good ad67ac46e35feab8c55a3e9c2331cab079212f35
# good: [3948c1ab72ed9331117926a67ab2e83e1a2b1d74] Merge staging-next into staging
git bisect good 3948c1ab72ed9331117926a67ab2e83e1a2b1d74
# good: [18c4006b6decdb2ee2de782fadbb9226238aba40] fast-float: 8.0.0 -> 8.0.1 (#388783)
git bisect good 18c4006b6decdb2ee2de782fadbb9226238aba40
# skip: [104dfc62a781776d3c4ca7aa322b41b6ed09d8be] freerdp3: 3.12.0 -> 3.14.0 (#387596)
git bisect skip 104dfc62a781776d3c4ca7aa322b41b6ed09d8be
# skip: [9748434b0b3de15d760cf1da9487c626c742da85] python313Packages.pytest: 8.3.4 -> 8.3.5
git bisect skip 9748434b0b3de15d760cf1da9487c626c742da85
# bad: [a6ffc120d2a2db51002cac0c3b05f76bb88b8c90] Merge staging-next into staging
git bisect bad a6ffc120d2a2db51002cac0c3b05f76bb88b8c90
```

In particular, replacing `gcc` with `clang` still gives the same build error.

```diff
diff --git a/pkgs/by-name/te/tectonic-unwrapped/package.nix b/pkgs/by-name/te/tectonic-unwrapped/package.nix
index 50d64c8416e1..11db053fe3dd 100644
--- a/pkgs/by-name/te/tectonic-unwrapped/package.nix
+++ b/pkgs/by-name/te/tectonic-unwrapped/package.nix
@@ -10,6 +10,10 @@
   lib,
   clangStdenv,
   fetchFromGitHub,
+  pkgsBuildHost,
+  pkgsBuildTarget,
+  pkgsTargetTarget,
+  rust,
   rustPlatform,
   fontconfig,
   harfbuzzFull,
@@ -25,6 +29,21 @@ let
     # use clang to work around build failure with GCC 14
     # see: https://github.com/tectonic-typesetting/tectonic/issues/1263
     stdenv = clangStdenv;
+    cargoBuildHook = rustPlatform.cargoBuildHook.override {
+      stdenv = clangStdenv;
+      rust = rust.override {
+        pkgsBuildHost = pkgsBuildHost // {
+          stdenv = clangStdenv;
+        };
+        pkgsBuildTarget = pkgsBuildTarget // {
+          stdenv = clangStdenv;
+        };
+        pkgsTargetTarget = pkgsTargetTarget // {
+          stdenv = clangStdenv;
+        };
+        stdenv = clangStdenv;
+      };
+    };
   };
 
 in
```

<details>
<summary>build log</summary>

```text
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/wm1k8y42lw9v17z2llypkh7ad3ah6dvy-source
source root is source
Executing cargoSetupPostUnpackHook
Finished cargoSetupPostUnpackHook
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
applying patch /nix/store/h1gxn7d6p4gx6gxiz8jsmzwr6lhwbvd0-1202-fix-build-with-rust-1_80
patching file Cargo.lock
Hunk #1 succeeded at 2359 (offset -274 lines).
Hunk #2 succeeded at 2683 (offset -252 lines).
Hunk #3 succeeded at 2706 (offset -252 lines).
patching file Cargo.toml
Hunk #1 succeeded at 97 with fuzz 2.
applying patch /nix/store/h4jarfpjv11pniijqamirda2d96h9c8k-1155-fix-endless-reruns-when-generating-bbl
patching file crates/engine_bibtex/src/lib.rs
Executing cargoSetupPostPatchHook
Validating consistency between /build/source/Cargo.lock and /build/tectonic-0.15.0-vendor/Cargo.lock
Finished cargoSetupPostPatchHook
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Executing cargoBuildHook
cargoBuildHook flags: -j 8 --target x86_64-unknown-linux-gnu --offline --profile release --features=external-harfbuzz
[1m[32m   Compiling[0m libc v0.2.153
[1m[32m   Compiling[0m proc-macro2 v1.0.78
[1m[32m   Compiling[0m unicode-ident v1.0.12
[1m[32m   Compiling[0m cfg-if v1.0.0
[1m[32m   Compiling[0m pkg-config v0.3.29
[1m[32m   Compiling[0m vcpkg v0.2.15
[1m[32m   Compiling[0m autocfg v1.1.0
[1m[32m   Compiling[0m thiserror v1.0.56
[1m[32m   Compiling[0m memchr v2.7.1
[1m[32m   Compiling[0m once_cell v1.19.0
[1m[32m   Compiling[0m version_check v0.9.4
[1m[32m   Compiling[0m log v0.4.20
[1m[32m   Compiling[0m tectonic_dep_support v0.1.1 (/build/source/crates/dep_support)
[1m[32m   Compiling[0m bitflags v2.4.2
[1m[32m   Compiling[0m itoa v1.0.10
[1m[32m   Compiling[0m regex-syntax v0.8.2
[1m[32m   Compiling[0m pin-project-lite v0.2.13
[1m[32m   Compiling[0m aho-corasick v1.1.2
[1m[32m   Compiling[0m pin-utils v0.1.0
[1m[32m   Compiling[0m same-file v1.0.6
[1m[32m   Compiling[0m quote v1.0.35
[1m[32m   Compiling[0m walkdir v2.4.0
[1m[32m   Compiling[0m bytes v1.5.0
[1m[32m   Compiling[0m syn v2.0.48
[1m[32m   Compiling[0m cc v1.0.83
[1m[32m   Compiling[0m getrandom v0.2.12
[1m[32m   Compiling[0m mio v0.8.10
[1m[32m   Compiling[0m rand_core v0.6.4
[1m[32m   Compiling[0m typenum v1.17.0
[1m[32m   Compiling[0m futures-core v0.3.30
[1m[32m   Compiling[0m ppv-lite86 v0.2.17
[1m[32m   Compiling[0m siphasher v0.3.11
[1m[32m   Compiling[0m phf_shared v0.11.2
[1m[32m   Compiling[0m rand_chacha v0.3.1
[1m[32m   Compiling[0m regex-automata v0.4.5
[1m[32m   Compiling[0m num_cpus v1.16.0
[1m[32m   Compiling[0m socket2 v0.5.5
[1m[32m   Compiling[0m signal-hook-registry v1.4.1
[1m[32m   Compiling[0m generic-array v0.14.7
[1m[32m   Compiling[0m slab v0.4.9
[1m[32m   Compiling[0m futures-sink v0.3.30
[1m[32m   Compiling[0m rand v0.8.5
[1m[32m   Compiling[0m num-traits v0.2.17
[1m[32m   Compiling[0m crossbeam-utils v0.8.19
[1m[32m   Compiling[0m tinyvec_macros v0.1.1
[1m[32m   Compiling[0m adler v1.0.2
[1m[32m   Compiling[0m anyhow v1.0.79
[1m[32m   Compiling[0m serde v1.0.196
[1m[32m   Compiling[0m bitflags v1.3.2
[1m[32m   Compiling[0m phf_generator v0.11.2
[1m[32m   Compiling[0m miniz_oxide v0.7.2
[1m[32m   Compiling[0m tinyvec v1.6.0
[1m[32m   Compiling[0m futures-channel v0.3.30
[1m[32m   Compiling[0m libz-sys v1.1.15
[1m[32m   Compiling[0m futures-io v0.3.30
[1m[32m   Compiling[0m crc32fast v1.3.2
[1m[32m   Compiling[0m futures-task v0.3.30
[1m[32m   Compiling[0m rustix v0.38.31
[1m[32m   Compiling[0m bstr v1.9.0
[1m[32m   Compiling[0m unicode-normalization v0.1.22
[1m[32m   Compiling[0m phf_codegen v0.11.2
[1m[32m   Compiling[0m thiserror-impl v1.0.56
[1m[32m   Compiling[0m tokio-macros v2.2.0
[1m[32m   Compiling[0m serde_derive v1.0.196
[1m[32m   Compiling[0m futures-macro v0.3.30
[1m[32m   Compiling[0m tectonic_bridge_core v0.4.1 (/build/source/crates/bridge_core)
[1m[32m   Compiling[0m linux-raw-sys v0.4.13
[1m[32m   Compiling[0m tokio v1.36.0
[1m[32m   Compiling[0m fnv v1.0.7
[1m[32m   Compiling[0m lazy_static v1.4.0
[1m[32m   Compiling[0m smallvec v1.13.1
[1m[32m   Compiling[0m futures-util v0.3.30
[1m[32m   Compiling[0m hex v0.4.3
[1m[32m   Compiling[0m tectonic_cfg_support v0.1.4 (/build/source/crates/cfg_support)
[1m[32m   Compiling[0m gix-hash v0.11.4
[1m[32m   Compiling[0m tracing-attributes v0.1.27
[1m[32m   Compiling[0m tectonic_errors v0.2.1 (/build/source/crates/errors)
[1m[32m   Compiling[0m crypto-common v0.1.6
[1m[32m   Compiling[0m block-buffer v0.10.4
[1m[32m   Compiling[0m tracing-core v0.1.32
[1m[32m   Compiling[0m sha1_smol v1.0.0
[1m[32m   Compiling[0m gix-features v0.29.0
[1m[32m   Compiling[0m tracing v0.1.40
[1m[32m   Compiling[0m digest v0.10.7
[1m[32m   Compiling[0m flate2 v1.0.28
[1m[32m   Compiling[0m openssl-sys v0.9.99
[1m[32m   Compiling[0m lock_api v0.4.11
[1m[32m   Compiling[0m minimal-lexical v0.2.1
[1m[32m   Compiling[0m fastrand v2.0.1
[1m[32m   Compiling[0m home v0.5.9
[1m[32m   Compiling[0m time-core v0.1.2
[1m[32m   Compiling[0m powerfmt v0.2.0
[1m[32m   Compiling[0m parking_lot_core v0.9.9
[1m[32m   Compiling[0m num-conv v0.1.0
[1m[32m   Compiling[0m nom v7.1.3
[1m[32m   Compiling[0m time-macros v0.2.18
[1m[32m   Compiling[0m deranged v0.3.11
[1m[32m   Compiling[0m tectonic_status_base v0.2.1 (/build/source/crates/status_base)
[1m[32m   Compiling[0m memoffset v0.7.1
[1m[32m   Compiling[0m scopeguard v1.2.0
[1m[32m   Compiling[0m num_threads v0.1.6
[1m[32m   Compiling[0m equivalent v1.0.1
[1m[32m   Compiling[0m xdg v2.5.2
[1m[32m   Compiling[0m cpufeatures v0.2.12
[1m[32m   Compiling[0m hashbrown v0.14.3
[1m[32m   Compiling[0m unicode-width v0.1.11
[1m[32m   Compiling[0m sha2 v0.10.8
[1m[32m   Compiling[0m time v0.3.36
[1m[32m   Compiling[0m indexmap v2.2.2
[1m[32m   Compiling[0m app_dirs2 v2.5.5
[1m[32m   Compiling[0m tempfile v3.9.0
[1m[32m   Compiling[0m crossbeam-epoch v0.9.18
[1m[32m   Compiling[0m tectonic_bridge_graphite2 v0.2.2 (/build/source/crates/bridge_graphite2)
[1m[32m   Compiling[0m gix-trace v0.1.7
[1m[32m   Compiling[0m tectonic_bridge_flate v0.1.7 (/build/source/crates/bridge_flate)
[1m[32m   Compiling[0m gix-path v0.8.4
[1m[32m   Compiling[0m gix-date v0.5.1
[1m[32m   Compiling[0m crossbeam-deque v0.8.5
[1m[32m   Compiling[0m parking_lot v0.12.1
[1m[32m   Compiling[0m phf v0.11.2
[1m[32m   Compiling[0m tectonic_io_base v0.4.3 (/build/source/crates/io_base)
[1m[32m   Compiling[0m gix-fs v0.1.1
[1m[32m   Compiling[0m btoi v0.4.3
[1m[32m   Compiling[0m globset v0.4.14
[1m[32m   Compiling[0m foreign-types-shared v0.1.1
[1m[32m   Compiling[0m percent-encoding v2.3.1
[1m[32m   Compiling[0m openssl v0.10.63
[1m[32m   Compiling[0m ignore v0.4.22
[1m[32m   Compiling[0m foreign-types v0.3.2
[1m[32m   Compiling[0m gix-actor v0.20.0
[1m[32m   Compiling[0m gix-tempfile v5.0.3
[1m[32m   Compiling[0m nix v0.26.4
[1m[32m   Compiling[0m regex v1.10.3
[1m[32m   Compiling[0m gix-utils v0.1.9
[1m[32m   Compiling[0m md-5 v0.10.6
[1m[32m   Compiling[0m tectonic_pdf_io v0.4.1 (/build/source/crates/pdf_io)
[1m[32m   Compiling[0m gix-validate v0.7.7
[1m[32m   Compiling[0m http v0.2.11
[1m[32m   Compiling[0m openssl-macros v0.1.1
[1m[32m   Compiling[0m miette-derive v5.10.0
[1m[32m   Compiling[0m tectonic_bridge_harfbuzz v0.2.9 (/build/source/crates/bridge_harfbuzz)
[1m[32m   Compiling[0m tectonic_bridge_icu v0.2.1 (/build/source/crates/bridge_icu)
[1m[32m   Compiling[0m tectonic_bridge_freetype2 v0.2.0 (/build/source/crates/bridge_freetype2)
[1m[32m   Compiling[0m ryu v1.0.16
[1m[32m   Compiling[0m native-tls v0.2.11
[1m[32m   Compiling[0m httparse v1.8.0
[1m[32m   Compiling[0m gix-object v0.29.2
[1m[32m   Compiling[0m miette v5.10.0
[1m[32m   Compiling[0m gix-lock v5.0.1
[1m[32m   Compiling[0m parse-zoneinfo v0.3.0
[1m[32m   Compiling[0m tokio-util v0.7.10
[1m[32m   Compiling[0m futures-executor v0.3.30
[1m[32m   Compiling[0m terminfo v0.8.0
[1m[32m   Compiling[0m dirs-sys v0.3.7
[1m[32m   Compiling[0m inotify-sys v0.1.5
[1m[32m   Compiling[0m memmap2 v0.5.10
[1m[32m   Compiling[0m openssl-probe v0.1.5
[1m[32m   Compiling[0m ucd-trie v0.1.6
[1m[32m   Compiling[0m try-lock v0.2.5
[1m[32m   Compiling[0m async-trait v0.1.77
[1m[32m   Compiling[0m want v0.3.1
[1m[32m   Compiling[0m pest v2.7.6
[1m[32m   Compiling[0m gix-ref v0.29.1
[1m[32m   Compiling[0m dirs v4.0.0
[1m[32m   Compiling[0m inotify v0.9.6
[1m[32m   Compiling[0m chrono-tz-build v0.2.1
[1m[32m   Compiling[0m h2 v0.3.24
[1m[32m   Compiling[0m futures v0.3.30
[1m[32m   Compiling[0m http-body v0.4.6
[1m[32m   Compiling[0m form_urlencoded v1.2.1
[1m[32m   Compiling[0m gix-glob v0.7.0
[1m[32m   Compiling[0m gix-config-value v0.12.5
[1m[32m   Compiling[0m tokio-stream v0.1.14
[1m[32m   Compiling[0m tectonic_xetex_layout v0.2.4 (/build/source/crates/xetex_layout)
[1m[32m   Compiling[0m nibble_vec v0.1.0
[1m[32m   Compiling[0m crossbeam-channel v0.5.11
[1m[32m   Compiling[0m filetime v0.2.23
[1m[32m   Compiling[0m gix-sec v0.8.4
[1m[32m   Compiling[0m proc-macro-error-attr v1.0.4
[1m[32m   Compiling[0m httpdate v1.0.3
[1m[32m   Compiling[0m libm v0.2.8
[1m[32m   Compiling[0m unicode-bidi v0.3.15
[1m[32m   Compiling[0m syn v1.0.109
[1m[32m   Compiling[0m endian-type v0.1.2
[1m[32m   Compiling[0m either v1.9.0
[1m[32m   Compiling[0m unicode-bom v2.0.3
[1m[32m   Compiling[0m unic-common v0.9.0
[1m[32m   Compiling[0m tower-service v0.3.2
[1m[32m   Compiling[0m unic-char-range v0.9.0
[1m[32m   Compiling[0m unic-ucd-version v0.9.0
[1m[32m   Compiling[0m gix-config v0.22.0
[1m[32m   Compiling[0m which v4.4.2
[1m[32m   Compiling[0m unic-char-property v0.9.0
[1m[32m   Compiling[0m idna v0.5.0
[1m[32m   Compiling[0m radix_trie v0.2.1
[1m[32m   Compiling[0m pest_meta v2.7.6
[1m[32m   Compiling[0m notify v5.2.0
[1m[32m   Compiling[0m hyper v0.14.28
[1m[32m   Compiling[0m project-origins v1.2.0
[1m[32m   Compiling[0m chrono-tz v0.8.5
[1m[32m   Compiling[0m tokio-native-tls v0.3.1
[1m[32m   Compiling[0m watchexec-signals v1.0.0
[1m[32m   Compiling[0m proc-macro-error v1.0.4
[1m[32m   Compiling[0m serde_json v1.0.113
[1m[32m   Compiling[0m event-listener v2.5.3
[1m[32m   Compiling[0m dunce v1.0.4
[1m[32m   Compiling[0m byteorder v1.5.0
[1m[32m   Compiling[0m base64 v0.21.7
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: clang++: warning: no such include directory: '/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include' [-Wmissing-include-dirs]
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: clang++: warning: no such include directory: '/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include' [-Wmissing-include-dirs]
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: clang++: warning: no such include directory: '/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include' [-Wmissing-include-dirs]
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: In file included from layout/xetex-XeTeXLayoutInterface.cpp:36:
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: In file included from /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/ubidi.h:26:
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:561:26: error: 'auto' not allowed in template parameter until C++17
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   561 | template <typename Type, auto closeFunction>
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                          ^~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: In file included from layout/xetex-XeTeXLayoutInterface.cpp:36:
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/ubidi.h:579:55: error: value of type 'void (UBiDi *)' is not implicitly convertible to 'int'
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   579 | U_DEFINE_LOCAL_OPEN_POINTER(LocalUBiDiPointer, UBiDi, ubidi_close);
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                                                       ^~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/urename.h:436:21: note: expanded from macro 'ubidi_close'
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   436 | #define ubidi_close U_ICU_ENTRY_POINT_RENAME(ubidi_close)
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/uvernum.h:123:47: note: expanded from macro 'U_ICU_ENTRY_POINT_RENAME'
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   123 | #       define U_ICU_ENTRY_POINT_RENAME(x)    U_DEF2_ICU_ENTRY_POINT_RENAME(x,U_ICU_VERSION_SUFFIX)
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/uvernum.h:122:51: note: expanded from macro 'U_DEF2_ICU_ENTRY_POINT_RENAME'
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   122 | #       define U_DEF2_ICU_ENTRY_POINT_RENAME(x,y) U_DEF_ICU_ENTRY_POINT_RENAME(x,y)
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/uvernum.h:121:50: note: expanded from macro 'U_DEF_ICU_ENTRY_POINT_RENAME'
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   121 | #       define U_DEF_ICU_ENTRY_POINT_RENAME(x,y) x ## y
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                                                  ^~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: <scratch space>:287:1: note: expanded from here
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   287 | ubidi_close_76
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       | ^~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:551:68: note: expanded from macro 'U_DEFINE_LOCAL_OPEN_POINTER'
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:   551 |     using LocalPointerClassName = internal::LocalOpenPointer<Type, closeFunction>
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |                                                                    ^~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: layout/xetex-XeTeXLayoutInterface.cpp:1135:12: warning: returning address of local temporary object [-Wreturn-stack-address]
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:  1135 |     return XeTeXFontMgr::GetFontManager()->getPlatformFontDesc(fontRef).c_str();
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4:       |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_xetex_layout@0.2.4: 1 warning and 2 errors generated.
[1m[31merror[0m[1m:[0m failed to run custom build command for `tectonic_xetex_layout v0.2.4 (/build/source/crates/xetex_layout)`

Caused by:
  process didn't exit successfully: `/build/source/target/release/build/tectonic_xetex_layout-25215a43ebd3f08c/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-env-changed=TECTONIC_DEP_BACKEND
  cargo:rerun-if-env-changed=TECTONIC_PKGCONFIG_FORCE_SEMI_STATIC
  cargo:rerun-if-env-changed=FONTCONFIG_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=SYSROOT
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-changed=layout/xetex-XeTeXFontInst.cpp
  cargo:rerun-if-changed=layout/xetex-XeTeXFontMgr.cpp
  cargo:rerun-if-changed=layout/xetex-XeTeXLayoutInterface.cpp
  cargo:rerun-if-changed=layout/xetex-XeTeXFontMgr_FC.cpp
  TARGET = Some("x86_64-unknown-linux-gnu")
  OPT_LEVEL = Some("3")
  HOST = Some("x86_64-unknown-linux-gnu")
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some("false")
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  cargo:rerun-if-env-changed=CXX_x86_64-unknown-linux-gnu
  CXX_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXX_x86_64_unknown_linux_gnu
  CXX_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = Some("/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-unknown-linux-gnu
  CXXFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_unknown_linux_gnu
  CXXFLAGS_x86_64_unknown_linux_gnu = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  running: "/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "layout" "-I" "/build/source/crates/bridge_core/support" "-I" "/nix/store/nziailpyy0d90vq5pi1kckbxzvjcg5ng-fontconfig-2.16.0-dev/include" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/r5iajf7yh32jw3c0hmnw0iwyx5lxhky0-harfbuzz-10.2.0-dev/include/harfbuzz" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include" "-I" "/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include" "-Wall" "-Wextra" "-Wall" "-std=c++14" "-Wall" "-Wdate-time" "-Wendif-labels" "-Wextra" "-Wformat=2" "-Wmissing-declarations" "-Wmissing-include-dirs" "-Wpointer-arith" "-Wredundant-decls" "-Wshadow" "-Wswitch-bool" "-Wundef" "-Wextra-semi" "-Wno-unused-parameter" "-Wno-implicit-fallthrough" "-fno-exceptions" "-fno-rtti" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/tectonic_xetex_layout-a8004e69e698997a/out/layout/xetex-XeTeXFontInst.o" "-c" "layout/xetex-XeTeXFontInst.cpp"
  cargo:warning=clang++: warning: no such include directory: '/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include' [-Wmissing-include-dirs]

  exit status: 0
  running: "/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "layout" "-I" "/build/source/crates/bridge_core/support" "-I" "/nix/store/nziailpyy0d90vq5pi1kckbxzvjcg5ng-fontconfig-2.16.0-dev/include" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/r5iajf7yh32jw3c0hmnw0iwyx5lxhky0-harfbuzz-10.2.0-dev/include/harfbuzz" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include" "-I" "/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include" "-Wall" "-Wextra" "-Wall" "-std=c++14" "-Wall" "-Wdate-time" "-Wendif-labels" "-Wextra" "-Wformat=2" "-Wmissing-declarations" "-Wmissing-include-dirs" "-Wpointer-arith" "-Wredundant-decls" "-Wshadow" "-Wswitch-bool" "-Wundef" "-Wextra-semi" "-Wno-unused-parameter" "-Wno-implicit-fallthrough" "-fno-exceptions" "-fno-rtti" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/tectonic_xetex_layout-a8004e69e698997a/out/layout/xetex-XeTeXFontMgr.o" "-c" "layout/xetex-XeTeXFontMgr.cpp"
  cargo:warning=clang++: warning: no such include directory: '/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include' [-Wmissing-include-dirs]

  exit status: 0
  running: "/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "layout" "-I" "/build/source/crates/bridge_core/support" "-I" "/nix/store/nziailpyy0d90vq5pi1kckbxzvjcg5ng-fontconfig-2.16.0-dev/include" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/r5iajf7yh32jw3c0hmnw0iwyx5lxhky0-harfbuzz-10.2.0-dev/include/harfbuzz" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include" "-I" "/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include" "-Wall" "-Wextra" "-Wall" "-std=c++14" "-Wall" "-Wdate-time" "-Wendif-labels" "-Wextra" "-Wformat=2" "-Wmissing-declarations" "-Wmissing-include-dirs" "-Wpointer-arith" "-Wredundant-decls" "-Wshadow" "-Wswitch-bool" "-Wundef" "-Wextra-semi" "-Wno-unused-parameter" "-Wno-implicit-fallthrough" "-fno-exceptions" "-fno-rtti" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/tectonic_xetex_layout-a8004e69e698997a/out/layout/xetex-XeTeXLayoutInterface.o" "-c" "layout/xetex-XeTeXLayoutInterface.cpp"
  cargo:warning=clang++: warning: no such include directory: '/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include' [-Wmissing-include-dirs]

  cargo:warning=In file included from layout/xetex-XeTeXLayoutInterface.cpp:36:

  cargo:warning=In file included from /nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/ubidi.h:26:

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:561:26: error: 'auto' not allowed in template parameter until C++17

  cargo:warning=  561 | template <typename Type, auto closeFunction>

  cargo:warning=      |                          ^~~~

  cargo:warning=In file included from layout/xetex-XeTeXLayoutInterface.cpp:36:

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/ubidi.h:579:55: error: value of type 'void (UBiDi *)' is not implicitly convertible to 'int'

  cargo:warning=  579 | U_DEFINE_LOCAL_OPEN_POINTER(LocalUBiDiPointer, UBiDi, ubidi_close);

  cargo:warning=      |                                                       ^~~~~~~~~~~

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/urename.h:436:21: note: expanded from macro 'ubidi_close'

  cargo:warning=  436 | #define ubidi_close U_ICU_ENTRY_POINT_RENAME(ubidi_close)

  cargo:warning=      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/uvernum.h:123:47: note: expanded from macro 'U_ICU_ENTRY_POINT_RENAME'

  cargo:warning=  123 | #       define U_ICU_ENTRY_POINT_RENAME(x)    U_DEF2_ICU_ENTRY_POINT_RENAME(x,U_ICU_VERSION_SUFFIX)

  cargo:warning=      |                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/uvernum.h:122:51: note: expanded from macro 'U_DEF2_ICU_ENTRY_POINT_RENAME'

  cargo:warning=  122 | #       define U_DEF2_ICU_ENTRY_POINT_RENAME(x,y) U_DEF_ICU_ENTRY_POINT_RENAME(x,y)

  cargo:warning=      |                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/uvernum.h:121:50: note: expanded from macro 'U_DEF_ICU_ENTRY_POINT_RENAME'

  cargo:warning=  121 | #       define U_DEF_ICU_ENTRY_POINT_RENAME(x,y) x ## y

  cargo:warning=      |                                                  ^~~~~~

  cargo:warning=<scratch space>:287:1: note: expanded from here

  cargo:warning=  287 | ubidi_close_76

  cargo:warning=      | ^~~~~~~~~~~~~~

  cargo:warning=/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include/unicode/localpointer.h:551:68: note: expanded from macro 'U_DEFINE_LOCAL_OPEN_POINTER'

  cargo:warning=  551 |     using LocalPointerClassName = internal::LocalOpenPointer<Type, closeFunction>

  cargo:warning=      |                                                                    ^~~~~~~~~~~~~

  cargo:warning=layout/xetex-XeTeXLayoutInterface.cpp:1135:12: warning: returning address of local temporary object [-Wreturn-stack-address]

  cargo:warning= 1135 |     return XeTeXFontMgr::GetFontManager()->getPlatformFontDesc(fontRef).c_str();

  cargo:warning=      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  cargo:warning=1 warning and 2 errors generated.

  exit status: 1

  --- stderr


  error occurred: Command "/nix/store/q87df3dw0x7r1k9gagir9a5sdi3gmw4a-clang-wrapper-19.1.7/bin/c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "layout" "-I" "/build/source/crates/bridge_core/support" "-I" "/nix/store/nziailpyy0d90vq5pi1kckbxzvjcg5ng-fontconfig-2.16.0-dev/include" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/r5iajf7yh32jw3c0hmnw0iwyx5lxhky0-harfbuzz-10.2.0-dev/include/harfbuzz" "-I" "/nix/store/2dgzl64w68mglcn6mi82rnvk37df33qz-freetype-2.13.3-dev/include/freetype2" "-I" "/nix/store/mfhahm2jw2x3nliqphcc4cc0f0w538yf-graphite2-1.3.14/include" "-I" "/nix/store/3xiw2ng0hg6ck7mzir3wpa1vh6vhsisb-icu4c-76.1-dev/include" "-Wall" "-Wextra" "-Wall" "-std=c++14" "-Wall" "-Wdate-time" "-Wendif-labels" "-Wextra" "-Wformat=2" "-Wmissing-declarations" "-Wmissing-include-dirs" "-Wpointer-arith" "-Wredundant-decls" "-Wshadow" "-Wswitch-bool" "-Wundef" "-Wextra-semi" "-Wno-unused-parameter" "-Wno-implicit-fallthrough" "-fno-exceptions" "-fno-rtti" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/tectonic_xetex_layout-a8004e69e698997a/out/layout/xetex-XeTeXLayoutInterface.o" "-c" "layout/xetex-XeTeXLayoutInterface.cpp" with args "c++" did not execute successfully (status code exit status: 1).


[1m[33mwarning[0m[1m:[0m build failed, waiting for other jobs to finish...
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-cidtype2.c:477:14: warning: variable 'num_glyphs' set but not used [-Wunused-but-set-variable]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   477 |     USHORT   num_glyphs;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |              ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-cidtype2.c:818:21: warning: variable 'gid' is used uninitialized whenever 'if' condition is true [-Wsometimes-uninitialized]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   818 |                 if (code < 0) {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |                     ^~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-cidtype2.c:839:17: note: uninitialized use occurs here
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   839 |             if (gid == 0) {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |                 ^~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-cidtype2.c:818:17: note: remove the 'if' if its condition is always false
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   818 |                 if (code < 0) {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |                 ^~~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   819 |                     dpx_warning("Unable to map CID to code: CID=%u", cid);
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   820 |                 } else {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |                 ~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-cidtype2.c:794:25: note: initialize the variable 'gid' to silence this warning
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   794 |             uint16_t gid;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |                         ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |                          = 0
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: 2 warnings generated.
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-mpost.c:1510:3: warning: ignoring return value of function declared with 'warn_unused_result' attribute [-Wunused-result]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:  1510 |   fread(buffer, sizeof(char), size, image_file);
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |   ^~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: 1 warning generated.
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-pdfcolor.c:370:11: warning: no previous extern declaration for non-static variable 'current_fill' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   370 | pdf_color current_fill   = {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |           ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-pdfcolor.c:370:1: note: declare 'static' if the variable is not intended to be used outside of this translation unit
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   370 | pdf_color current_fill   = {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       | ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-pdfcolor.c:379:11: warning: no previous extern declaration for non-static variable 'current_stroke' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   379 | pdf_color current_stroke = {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |           ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-pdfcolor.c:379:1: note: declare 'static' if the variable is not intended to be used outside of this translation unit
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   379 | pdf_color current_stroke = {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       | ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-pdfcolor.c:423:11: warning: no previous extern declaration for non-static variable 'default_color' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   423 | pdf_color default_color = {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |           ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-pdfcolor.c:423:1: note: declare 'static' if the variable is not intended to be used outside of this translation unit
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   423 | pdf_color default_color = {
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       | ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: 3 warnings generated.
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_html.c:98:5: warning: misleading indentation; statement is not part of the previous 'for' [-Wmisleading-indentation]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:    98 |     k = v = NULL;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |     ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_html.c:96:3: note: previous statement is here
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:    96 |   for (p = *pp ; p < endptr && isspace((unsigned char)*p); p++)
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |   ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: 1 warning generated.
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_misc.c:363:18: warning: no previous extern declaration for non-static variable 'fontattrs' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   363 | struct fontattr *fontattrs = NULL;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |                  ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_misc.c:363:1: note: declare 'static' if the variable is not intended to be used outside of this translation unit
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   363 | struct fontattr *fontattrs = NULL;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       | ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_misc.c:364:5: warning: no previous extern declaration for non-static variable 'num_fontattrs' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   364 | int num_fontattrs = 0;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |     ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_misc.c:364:1: note: declare 'static' if the variable is not intended to be used outside of this translation unit
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   364 | int num_fontattrs = 0;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       | ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_misc.c:365:5: warning: no previous extern declaration for non-static variable 'max_fontattrs' [-Wmissing-variable-declarations]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   365 | int max_fontattrs = 0;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |     ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_misc.c:365:1: note: declare 'static' if the variable is not intended to be used outside of this translation unit
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   365 | int max_fontattrs = 0;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       | ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: 3 warnings generated.
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-spc_util.c:1188:8: warning: missing field 'color' initializer [-Wmissing-field-initializers]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:  1188 |   {NULL}
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |        ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: 1 warning generated.
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-specials.c:615:8: warning: missing field 'bodhk_func' initializer [-Wmissing-field-initializers]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:   615 |   {NULL} /* end */
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |        ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: 1 warning generated.
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: pdf_io/dpx-tt_gsub.c:1935:11: warning: variable 'count' set but not used [-Wunused-but-set-variable]
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:  1935 |   int32_t count = 0;
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1:       |           ^
[1m[33mwarning[0m[1m:[0m tectonic_pdf_io@0.4.1: 1 warning generated.
```

</details>

--- a/pkgs/by-name/te/tectonic-unwrapped/package.nix
+++ b/pkgs/by-name/te/tectonic-unwrapped/package.nix
@@ -52,9 +52,9 @@ buildRustPackage rec {
   cargoPatches = [
     (fetchpatch2 {
       # cherry-picked from https://github.com/tectonic-typesetting/tectonic/pull/1202
-      name = "1202-fix-build-with-rust-1_80";
-      url = "https://github.com/tectonic-typesetting/tectonic/commit/6b49ca8db40aaca29cb375ce75add3e575558375.patch";
-      hash = "sha256-i1L3XaSuBbsmgOSXIWVqr6EHlHGs8A+6v06kJ3C50sk=";
+      name = "1202-fix-build-with-rust-1_80-and-icu-75";
+      url = "https://github.com/tectonic-typesetting/tectonic/compare/19654bf152d602995da970f6164713953cffc2e6...6b49ca8db40aaca29cb375ce75add3e575558375.patch?full_index=1";
+      hash = "sha256-CgQBCFvfYKKXELnR9fMDqmdq97n514CzGJ7EBGpghJc=";
     })
   ];
 

